src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -181,6 +181,8 @@ internal static class IDEDiagnosticIds
         public const string RemoveRedundantNullableDirectiveDiagnosticId = "IDE0240";
         public const string RemoveUnnecessaryNullableDirectiveDiagnosticId = "IDE0241";
 
+        public const string PreferTrailingCommaDiagnosticId = "IDE0250";
```

Comment:
💡 This seems unlikely to need its own group of IDs. Maybe move up to IDE0085?

Reply:
Fixed